help/de/Help_ch_GUIMenus.xml

```diff
@@ -178,27 +184,23 @@
                        srccredit="Christian Wehling" />
           </imageobject>
 
-          <textobject>
-            <phrase>Bildschirm Kontenhierarchie</phrase>
-          </textobject>
-
           <caption>
-            <para>Dies ist die Übersicht mit dem Beispiel einer <guilabel>Kontenstruktur</guilabel>.
+            <para>Dies ist das Beispiel einer <guilabel>Kontenübersicht</guilabel>.
             </para>
           </caption>
         </mediaobject>
       </screenshot>
     </figure>
 
     <sect2 id="AccTree-menus">
-      <title>Kontenhierarchie - Menüleiste</title>
+      <title>Kontenübersicht - Menüleiste</title>
 
-      <para>Die <firstterm>Menüleiste</firstterm> für das Fenster Kontenhierarchie enthält die nachfolgenden
+      <para>Die <firstterm>Menüleiste</firstterm> für die Kontenübersicht enthält die nachfolgenden
         aufgeführten Menüpunkte.
       </para>
 
       <sect3 id="AccTree-file-menu">
-        <title>Kontenhierarchie - <guimenu><accel>D</accel>atei</guimenu>-Menü</title>
+        <title>Kontenübersicht - <guimenu><accel>D</accel>atei</guimenu>-Menü</title>
 
         <para>In der folgenden Tabelle werden die Möglichkeiten im Menü Datei beschrieben.
```

Comment:
„Datei“ in Anführungszeichen?

Reply:
Oder dieses „guilabel“, von dem ich nicht weiß, wie es funktioniert?

Reply:
Jede andere Auszeichnung ist besser als Anführungszeichen. :-)
Siehe z.B. https://wiki.gnucash.org/wiki/GUI_Guidelines#Quoting_in_Strings
Hm, rgendwo hatten wir das doch auch für docbook …